docs/ThemeColors.stories.tsx

```diff
@@ -3,7 +3,8 @@ import tokens from '../src/figma/tokens.json';
 import { lightTheme, darkTheme } from '../src';
 import getCSSVariablesFromStylesheet from './utils/getCSSVariablesFromStylesheet';
 
-import { ColorSwatchGroup, ColorSwatch } from './components';
+import { ColorSwatch } from './components';
+import { ColorSwatchGroup } from './components/ColorSwatchGroup/deprecated';
```

Comment:
Since I am not doing theme colors in this PR, I made a deprecated version of ColorSwatchGroup so its not broken for now.